com.prateekmedia.appimagepool.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you submit the file upstream?

Reply:
I am the upstream, no currently appdata is not in upstream.

Reply:
so will it be included in the next release?

Reply:
Ok I will add it in the next release, I thought it was flatpak specific.